samples/client/others/rust/reqwest-oneof-discriminator/src/models/composed_disc_missing_from_properties.rs

```diff
@@ -0,0 +1,20 @@
+/*
+ * fruity
+ *
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * 
+ * Generated by: https://openapi-generator.tech
+ */
+
+
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "fruitType")]
+pub enum ComposedDiscMissingFromProperties {
```

Comment:
a bunch of these other `oneOf` types are still not being generated correctly, since the current implementation only supports mapped models:
https://github.com/OpenAPITools/openapi-generator/blob/ac78f1e85bb16f06b5531c527d9a1d64a2128d83/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java#L230-L243

For this PR, I just tried to fix what used to work. We might want to use `#[serde(untagged)]` enums for `oneOf` types that don't have discriminators, but that's a topic for another issue/PR :-)